runtime/JniInterface.cs

```diff
@@ -442,6 +442,10 @@ static class GlobalRefs
 
 		internal static object Unwrap(int i)
 		{
+			if (i == 0)
```

Comment:
I am really curious why they didn't make i an unsigned int, and require it to be > 0, since negative numbers on line 461 doesn't make sense.

Reply:
Actually, looking at the usage and when it was introduced https://github.com/Itiviti/ikvm/commit/398c051ca6d1017d67e48cb89a5d1bdb74cf2282
it looks like this was/is intended to be used by negative numbers, and it's gated by a `i < 0` check. 

I am not sure how we end up with 0 here.